src/Pos/DB/Block/Block.hs

```diff
@@ -0,0 +1,302 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Interface to Blocks DB.
+
+module Pos.DB.Block.Block
+       ( getBlock
+       , getBlockHeader
+       , getUndo
+       , getBlockWithUndo
+       , putBlund
+
+       , prepareBlockDB
+
+       -- * Load data
+       , loadBlundsWhile
+       , loadBlundsByDepth
+       , loadBlocksWhile
+       , loadHeadersWhile
+       , loadHeadersByDepth
+       , loadHeadersByDepthWhile
+       ) where
+
+import           Control.Lens              (_Wrapped)
+import           Control.Monad.Trans.Maybe (MaybeT (MaybeT), runMaybeT)
```

Comment:
Huh, that's weird. Okay, yes, let's export MaybeT and related functions from universum.